pkg/entity/data_connector/AzureBlobStorage.go

```diff
@@ -0,0 +1,58 @@
+package data_connector
```

Comment:
Please check the styleguide of Golang. I think file names should be snake cased.